exec-ui/src/app/modules/directory/components/portfolio-card/portfolio-card.component.scss

```diff
@@ -85,7 +85,7 @@
   .portfolio-avatar-circle {
     background-color: $avatar-404;
     height: 60px;
-    border: 1px solid $avatar-accent;
+    border: 1px solid $dark;
```

Comment:
Can you explain this style override? We want to keep it consistent across the site

Reply:
Reverted to original style